analyzer-comments/elm/top-scorers/use_foldl.md

```diff
@@ -0,0 +1,3 @@
+# use foldl and updateGoalCountForPlayer
```

Comment:
If this comment identical to `use_updateGoalCountForPlayer.md`, then I think it's wiser to only have one, and change the rule to be 

```elm
aggregateScorersMustUseFoldlAndUpdateGoalCountForPlayer : Rule
aggregateScorersMustUseFoldlAndUpdateGoalCountForPlayer =
    Analyzer.functionCalls
        { calledFrom = TopFunction "aggregateScorers"
        , findFunctions = [ FromExternalModule [ "List" ] "foldl", FromSameModule "updateGoalCountForPlayer"  ]
        , find = All
        , comment = Comment "Doesn't use List.foldl or updateGoalCountForPlayer" "elm.top-scorers.use_foldl" Essential Dict.empty
        }
```